be/src/storage/persistent_index.h

```diff
@@ -374,6 +374,7 @@ class ShardByLengthMutableIndex {
     uint32_t _fixed_key_size = -1;
     uint64_t _offset = 0;
     uint64_t _page_size = 0;
+    uint32_t _crc32c = 0;
```

Comment:
`_checksum` is a better name. You can change the algorithm to check it.

Reply:
Done